### DISCUSSION TITLE
Google Service Framework dependency

### DISCUSSION THREAD

**Discussion by n4b7l** [osmandapp/OsmAnd#17441] [Q&A]
Just wondering is it absolutely necessary to develop OsmAnd with Google Service Framework (GSF)? The only reason I use this map is to get rid of google tracking. If I am sending data to google one way or the other does not make sense. Please correct me if I am wrong.

**Comment by vshcherb** [comment]
Nope it's not needed as you can see there is a gradle target without google play services
